code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -577,7 +568,7 @@
 		holder.target = get_best_target(get_targets())
 	if(holder.target)
 		var/mob/living/M = holder.target
-		if(M && !istype(M.loc.type, /obj/icecube/flockdrone) && (M.getStatusDuration("stunned") || M.getStatusDuration("weakened") || M.getStatusDuration("paralysis") || M.stat))
+		if(M && !istype(M.loc, /obj/icecube/flockdrone) && (M.getStatusDuration("stunned") || M.getStatusDuration("weakened") || M.getStatusDuration("paralysis") || M.stat))
```

Comment:
Proc for all this iirc

Reply:
Can't find one, but I see a `#define` in the cyborg code for "am I stunned"

Reply:
it should be for all mobs

Reply:
hrm it's not, can you make this into one? put in helpers.dm, a #define works

Reply:
just the latter portion, not the ice cube stuff